### PR TITLE
[R3C-469] [MCH] Update Electronic Mapping from Pt2

### PR DESCRIPTION
**Pull request by aphecetche**
Add some more chambers and get the mapping in sync with the latest Pt2 changes.

### CODE REVIEW

File: Detectors/MUON/MCH/Raw/ElecMap/src/CH7L.cxx [hunk 1]
```diff
@@ -15,908 +15,907 @@
...
-  add(e2d, 719, 107, 860, 0, 1);
-  add(e2d, 719, 108, 860, 0, 2);
-  add(e2d, 719, 101, 860, 2, 0);
-  add(e2d, 719, 102, 860, 2, 1);
-  add(e2d, 719, 103, 860, 2, 2);
-  add(e2d, 719, 1134, 860, 4, 0);
-  add(e2d, 719, 1133, 860, 4, 1);
-  add(e2d, 719, 1129, 860, 6, 0);
-  add(e2d, 719, 1128, 860, 6, 1);
+  add(e2d, 707, 8, 720, 0, 0);
```
Comment (aferrero2707) [inline Detectors/MUON/MCH/Raw/ElecMap/src/CH7L.cxx:18]: Why this has changed? The new mapping looks correct while the old seems to be wrong, but why the old was not ok?

File: Detectors/MUON/MCH/Raw/ElecMap/src/CH7L.cxx [hunk 2]
```diff
@@ -15,908 +15,907 @@
...
+  add(e2d, 719, 102, 852, 3, 1);
+  add(e2d, 719, 103, 852, 3, 2);
+  add(e2d, 719, 1134, 852, 5, 0);
+  add(e2d, 719, 1133, 852, 5, 1);
+  add(e2d, 719, 1129, 852, 7, 0);
+  add(e2d, 719, 1128, 852, 7, 1);
 }
 void fillSolar2FeeLinkCH7L(std::map<uint16_t, uint32_t>& s2f)
 {
-  add_cru(s2f, 30, 0, 728);
```
Comment (aferrero2707) [inline Detectors/MUON/MCH/Raw/ElecMap/src/CH7L.cxx:884]: Link ID 728 belongs to CH8L... why in the old mapping is was associated to CH7L?

File: Detectors/MUON/MCH/Raw/ElecMap/src/CH7L.cxx [hunk 3]
```diff
@@ -15,908 +15,907 @@
...
-  add_cru(s2f, 32, 5, 306);
-  add_cru(s2f, 32, 6, 307);
-  add_cru(s2f, 32, 7, 308);
-  add_cru(s2f, 32, 8, 309);
-  add_cru(s2f, 32, 9, 856);
-  add_cru(s2f, 32, 10, 857);
-  add_cru(s2f, 32, 11, 858);
-  add_cru(s2f, 33, 0, 859);
-  add_cru(s2f, 33, 1, 860);
+  add_cru(s2f, 30, 0, 720);
```
Comment (aferrero2707) [inline Detectors/MUON/MCH/Raw/ElecMap/src/CH7L.cxx:883]: This is correct!

File: Detectors/MUON/MCH/Raw/ElecMap/src/elecmap.py [hunk 4]
```diff
@@ -87,7 +87,7 @@ def gencode_do(df, df_cru, solar_map, chamber):
     for row in df_cru.itertuples():
         if len(row.solar_id) > 0:
             out.write("add_cru(s2f,{},{},{});\n".format(
-                row.fee_id, row.link_id, row.solar_id))
+                row.fee_id, int(row.link_id)%12, row.solar_id))
```
Comment (aferrero2707) [inline Detectors/MUON/MCH/Raw/ElecMap/src/elecmap.py:90]: This is not strictly needed with the latest version of the mapping on the web, but we can keep it as an extra safety.

**Review by aferrero2707 — APPROVED** [review]

**Comment by aphecetche** [comment]
@aferrero2707 concerning your questions wrt to the old mapping, my answer is : I don't know :wink: 
I can only assume that last time the conversion script was ran the Pt2 sheet was indeed wrong ?

**Comment by aferrero2707** [comment]
Could be... do you remember when the previous mapping had been generated? I can check in the web sheet history.

**Comment by aphecetche** [comment]
I don't remember but according to GitHub the last changes are from Apr 5th 2020 I would say : https://github.com/AliceO2Group/AliceO2/pull/3283

**Comment by aferrero2707** [comment]
I can confirm that the google sheet on 5th of April 2020 was **wrong** for what concerns CH7L... in fact all ST4 was likely still work in progress.

**Comment by aphecetche** [comment]
@ktf @davidrohr @shahor02 sorry to bother you, but I'm again faced with the "issue" of not begin able to merge my own PRs. This one is green and has been approved (by @aferrero2707) : can someone please merge it (while it still is green :wink: ) ? thanks.

**Comment by davidrohr** [comment]
@aphecetche : I think you couldn't merge it since there was no real approval by aferrero2707, at least I can not see it in the conversation history of this PR.

**Comment by aphecetche** [comment]
@davidrohr well, I actually see a "aferrero2707 approved these changes 2 hours ago" in the conversation history and a check mark in the reviewers right column. But anyway, thanks for the merging !

**Comment by davidrohr** [comment]
ah, indeed, there was just some discussion afterwards. Perhaps you touched a CMake file for which his approval was not sufficient? Anyway, next time, when it's not urgent, we can try to understand before merging :)

**Pull request merged**